dataRetriever/requestHandlers/requestHandler.go

```diff
@@ -502,12 +532,8 @@ func (rrh *resolverRequestHandler) RequestTrieNode(requestHash []byte, topic str
 		"chunk index", chunkIndex,
 	)
 
-	requester, err := rrh.requestersFinder.MetaChainRequester(topic)
+	requester, err := getRequesterFunc(topic)
```

Comment:
can you make a better name for getRequesterFunc ?

Reply:
Right, renamed them